apps/libs/rootserver/src/index.ts

```diff
@@ -0,0 +1,17 @@
+import { ConfigService } from '@nestjs/config'
+import { FastifyAdapter, NestFastifyApplication } from '@nestjs/platform-fastify'
+
+export abstract class RootServer {
+  app?: NestFastifyApplication
+  adapter = new FastifyAdapter({ logger: true })
+
+  abstract create (): Promise<NestFastifyApplication>
+
+  abstract configure (app: NestFastifyApplication, config: ConfigService): Promise<void>
+
+  abstract init (app: NestFastifyApplication, config: ConfigService): Promise<void>
+
+  abstract start (): Promise<void>
+
+  abstract stop (): Promise<void>
```

Comment:
I think a few methods can be implemented since they are not shared?

Reply:
yea earlier i was thinking they might be vary in future and easy to fix... btw its' good to move them to root as well...